omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -182,11 +182,40 @@
 default['private_chef']['opscode-erchef']['bulk_fetch_batch_size'] = '5'
 default['private_chef']['opscode-erchef']['max_cache_size'] = '10000'
 default['private_chef']['opscode-erchef']['cache_ttl'] = '3600'
+default['private_chef']['opscode-erchef']['udp_socket_pool_size'] = '20'
+# Pool configuration for postgresql connections
+#
+# db_pool_size - the number of pgsql connections in the pool
+#
+# db_pool_queue_max - the maximum number of pgsql requests to queue up
+# if all connections are busy
+#
+# db_pooler_timeout - the maximum amount of time a request should wait
+# in the queue before timing out.  Request queueing is only effective
+# if db_pooler_timeout > 0
 default['private_chef']['opscode-erchef']['db_pool_size'] = '20'
 default['private_chef']['opscode-erchef']['db_pooler_timeout'] = '0'
 default['private_chef']['opscode-erchef']['sql_db_timeout'] = 5000
-default['private_chef']['opscode-erchef']['udp_socket_pool_size'] = '20'
 default['private_chef']['opscode-erchef']['db_pool_queue_max'] = '20'
+# Pool configuration for depsolver workers
+#
+# depsolver_worker_count - the number of depselector workers.  This is
+# a CPU bound task, thus setting this over the number of CPUs is not
+# advised.
+#
+# depsolver_pool_queue_max - the number of depsolve requets to queue
+# if all workers are busy.
+#
+# depsolver_pooler_timeout - the time in ms to wait before a queued
+# request times out.  Requesting queueing is only effective if
+# db_pooler_timeout > 0
+#
+# depsolver_tiemout - the amount of time to wait in ms before a
+# request to a depsolver worker is abandoned.
+default['private_chef']['opscode-erchef']['depsolver_pooler_timeout'] = '0'
+default['private_chef']['opscode-erchef']['depsolver_pool_queue_max'] = '50'
```

Comment:
Why are we, in two places, allowing a queue of 50 but setting a timeout of zero? It won't queue if the timeout is set to zero.